File: fib.java
import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of Fibonacci numbers to generate: ");
        int n = scanner.nextInt();

        int f1 = 0, f2 = 1, f3;
        System.out.print(f1 + " " + f2);

        for (int i = 2; i < n; i++) {
            f3 = f1 + f2;
            System.out.print(" " + f3);
            f1 = f2;
            f2 = f3;
        }
        System.out.println();
    }
}

File: fib.c
#include <stdio.h>
int main() {

  int i, n;

  int t1 = 0, t2 = 1;

  int n3 = t1 + t2;

  printf("Enter the number of terms: ");
  scanf("%d", &n);

  printf("Fibonacci Series: %d, %d, ", t1, t2);

  for (i = 3; i <= n; ++i) {
    printf("%d, ", n3);
    t1 = t2;
    t2 = n3;
    n3 = t1 + t2;
  }

  return 0;
}
